src/ai/susi/server/api/aaa/ResendVerificationLinkService.java

```diff
@@ -31,6 +31,9 @@
 
 /**
  * Created by dravit on 25/7/17.
+ * parameter : emailId
+ * sample request : 
+ * http://127.0.0.1:4000/aaa/resendVerificatiionLink.json?emailId=example@example.com
```

Comment:
I thought we just need to show examples hence used it as dummy data. I have updated it to a proper one now. Thanks for pointing out. There was another typo 'in resendVerificatiionLink'. fixed that as well.